ray-operator/config/samples/ray-service.autoscaler.yaml

```diff
@@ -0,0 +1,106 @@
+# Make sure to increase resource requests and limits before using this example in production.
+# For examples with more realistic resource configuration, see
+# ray-cluster.complete.large.yaml and
+# ray-cluster.autoscaler.large.yaml.
+apiVersion: ray.io/v1alpha1
+kind: RayService
+metadata:
+  name: rayservice-sample
+spec:
+  serviceUnhealthySecondThreshold: 300 # Config for the health check threshold for service. Default value is 60.
+  deploymentUnhealthySecondThreshold: 300 # Config for the health check threshold for deployments. Default value is 60.
+  serveConfig:
+    importPath: fruit.deployment_graph
+    runtimeEnv: |
+      working_dir: "https://github.com/ray-project/test_dag/archive/41d09119cbdf8450599f993f51318e9e27c59098.zip"
+    deployments:
+      - name: MangoStand
+        numReplicas: 1
+        userConfig: |
+          price: 3
+        rayActorOptions:
+          numCpus: 1
+      - name: OrangeStand
+        numReplicas: 1
+        userConfig: |
+          price: 2
+        rayActorOptions:
+          numCpus: 0.1
+      - name: PearStand
+        numReplicas: 1
+        userConfig: |
+          price: 1
+        rayActorOptions:
+          numCpus: 0.1
+      - name: FruitMarket
+        numReplicas: 1
+        rayActorOptions:
+          numCpus: 0.1
+      - name: DAGDriver
+        numReplicas: 1
+        routePrefix: "/"
+        rayActorOptions:
+          numCpus: 0.1
+  rayClusterConfig:
+    rayVersion: '2.100.0' # should match the Ray version in the image of the containers
```

Comment:
It looks like [the Ray images](https://github.com/ray-project/kuberay/pull/1037/files#diff-977cfa49bfd741168136aa416c0b8c400c93aa3fb9d8aaddef58d7bac0459e35R65) in this spec use Ray 2.4, but the Ray version here is set to `2.100.0` (which I assume is Ray 2.1).

Could we either change the Ray version or change the comments saying that the version should match the package in the images?

Reply:
Thanks! I have updated `rayVersion` to `2.4.0` https://github.com/ray-project/kuberay/pull/1037/commits/cc93f97bfc8071a7f14976c52d30221d2904c381. The behavior is the same as before. All values of `rayVersion` 2.X.Y have the same behavior.